etomica-core/src/main/java/etomica/nbr/list/NeighborListManager.java

```diff
@@ -417,7 +416,7 @@ public void addAtomNotify(IAtom atom) {
 
     public double getRange() {
         // TODO
```

Comment:
Is this still TODO?

Reply:
If we're satisfied that the whole setRange stack (pml -> nlm -> ncm -> lattice) works correctly then no.